Riot/Modules/MatrixKit/Models/Room/MXKRoomBubbleCellData.m

```diff
@@ -55,7 +55,22 @@ - (instancetype)initWithEvent:(MXEvent *)event andRoomState:(MXRoomState *)roomS
             [bubbleComponents addObject:firstComponent];
             
             senderId = event.sender;
-            targetId = [event.type isEqualToString:kMXEventTypeStringRoomMember] ? event.stateKey : nil;
+            if ([event.type isEqualToString:kMXEventTypeStringRoomMember])
+            {
+                NSString *membership = event.wireContent[@"membership"];
+                if (![membership isEqualToString:@"join"])
+                {
+                    targetId = event.stateKey;
+                }
+                else
+                {
+                    targetId = event.sender;
```

Comment:
I'm not sure I understand why this part is necessary. If the cell now always shows the sender avatar, then how come the `targetId` needs to be explicitly set to the sender here?

Reply:
You're right. I just found this bug while digging. I actually hesitated to add this fix and I can revert it if you want.

Reply:
@ismailgulek I agree. I've just checked with android people and they confirmed the behaviour is the same. I'll check with the design team in this case.

Reply:
@ismailgulek I've checked with @amshakal and we'll display the avatar of the invitee in the collapsed membership cell even if it's a bit inconsistent with the expanded membership.

Reply:
Great, thanks for checking.